### ISSUE TITLE
Compass does not compile @-moz-document url-prefix() correctly (sass does)

### ISSUE THREAD

**Issue by AyeAyeApi** [Compass/compass#1921]
It seems there is a bug in the way Compass handles `@-moz-document url-prefix()` that does not occur in sass.

Example scss:

``` css
#selectWrapper {
    select {
        width: 100%;
        @-moz-document url-prefix() {
            overflow: hidden;
            width: 165px;
        }
    }
}
```

Expected output (tested using http://sassmeister.com/):

``` css
#selectWrapper select {
    width: 100%;
}
@-moz-document url-prefix() {
    #selectWrapper select {
        overflow: hidden;
        width: 165px;
    }
}
```

Compass output:

``` css
#selectWrapper select {
    width: 100%;
    @-moz-document url-prefix() {
        overflow:hidden;
        width: 165 px
    }
}
```

I believe this was fixed in SASS 3.2 as per sass/libsass#633

Forgive any inaccuracies in my information, I'm a bit new to compass, but I couldn't find any mention of this in open issues.

**Comment by AyeAyeApi** [comment]
As a temporary fix for my own use case, I included a final `_compassHacks.scss` with the following:

``` css
@-moz-document url-prefix() {
    #selectWrapper select {
        overflow:hidden;
        width: 165 px
    }
}
```

**Comment by cimmanon** [comment]
Compass is just a wrapper around Sass.  If you're using an old version of Compass, you're going to be using an old version of Sass.  If the behavior is fixed in Sass 3.3+, then you need to be using Compass 1.0.

**Comment by AyeAyeApi** [comment]
Ah, you are correct. I still had 0.12.2, updating to 1.0.3 fixed the problem. Sorry about that. :)
